src/main.rs

```diff
@@ -0,0 +1,38 @@
+mod generate_value;
+use std::io;
+use std::io::Write;
+
+fn main(){
+    let result:u8 = generate_value::get_value();
+    let mut trying:u8 = 1;
```

Comment:
acho que essas tipagens inline não é necessário, talvez agora para a gente se acostumar é uma boa. Mas, futuramente, não precisa. Pois, ele implicitamente já está tipando a variavel quando está sendo atribuida, isso segue no padrão Typescript também.

```suggestion
    let result = generate_value::get_value();
    let mut trying = 1;
```

Reply:
especifiquei mais por conta do uso de memoria, por ser uma valor pequeno.

Reply:
verdade, tem isso também. Boa! Vou fazer o mesmo.